nifi-nar-bundles/nifi-marklogic-bundle/NOTICE.txt

```diff
@@ -0,0 +1,2106 @@
+MarkLogic NiFi Processors
+
+
+Copyright � 2018 MarkLogic Corporation.

```

Comment:
This notice file should be at the same location under META-INF just like this notice file is for the azure nar

https://github.com/apache/nifi/blob/master/nifi-nar-bundles/nifi-azure-bundle/nifi-azure-nar/src/main/resources/META-INF/NOTICE

Also, it isn't Copyright MarkLogic if you're submitting this as a pull request to the Apache Software Foundation.  The notice header then should read
---
nifi-marklogic-nar
Copyright 2015-2018 The Apache Software Foundation

This product includes software developed at
The Apache Software Foundation (http://www.apache.org/).
--

Then be followed by that which is legally required by the binary dependencies of this nar bundle - no more no less.